code/modules/power/singularity/narsie.dm

```diff
@@ -52,23 +53,9 @@
 		mezzer()
 
 
-/obj/singularity/narsie/Bump(atom/A)//you dare stand before a god?!
-	godsmack(A)
-	return
-
-/obj/singularity/narsie/Bumped(atom/A)
-	godsmack(A)
-	return
-
-/obj/singularity/narsie/proc/godsmack(atom/A)
-	if(isobj(A))
-		var/obj/O = A
-		O.ex_act(1)
-		if(O) qdel(O)
-
-	else if(isturf(A))
-		var/turf/T = A
-		T.ChangeTurf(/turf/simulated/floor/plasteel/cult)
+/obj/singularity/narsie/Bump(atom/A)
+	forceMove(get_turf(A))
```

Comment:
So it doesn't get stuck on stuff; there's probably a better way to do it but basically I want it to go through walls and shit